packages/fxa-content-server/app/tests/spec/lib/payment-server.js

```diff
@@ -32,12 +32,18 @@ describe('lib/payment-server-redirect', () => {
     view = {
       getSignedInAccount: sinon.stub().callsFake(() => account),
       navigateAway: sinon.spy(),
+      metrics: {
+        getFlowEventMetadata: sinon.spy(() => ({
+          flowBeginTime: 4321,
+          flowId: 'foo',
+        })),
+      },
     };
   });
 
   it('redirects as expected', () => {
     const REDIRECT_PATH = 'example/path';
-    PaymentServer.navigateToPaymentServer(
+    return PaymentServer.navigateToPaymentServer(
```

Comment:
These tests weren't returning the promises and were actually failing, so had to do a bit of tangential repair.

Reply:
(whoops!)